flask/src/server.py

```diff
@@ -3,39 +3,37 @@
 import os
 
 from girder_client import GirderClient
-from flask import Flask, send_file, request, Response
+from flask import Flask, request, Response, send_file
 from flask_cors import CORS
 
 app = Flask(__name__)
 
-CORS(app, expose_headers=["x-suggested-filename"])
+cors_route = os.environ.get('CORS_API', '/*')
```

Comment:
Its not the route we want to configure its the origin, where the request is coming from?